pkg/kv/kvserver/loqrecovery/apply.go

```diff
@@ -274,7 +274,8 @@ func applyReplicaUpdate(
 			Txn:    res.Intent.Txn,
 			Status: roachpb.ABORTED,
 		}
-		if _, _, _, err := storage.MVCCResolveWriteIntent(ctx, readWriter, &ms, update, storage.MVCCResolveWriteIntentOptions{}); err != nil {
+		// TODO(arul): figure out if this needs any handling.
```

Comment:
That's ok, we don't need to do anything special here. We resolve intent of the range descriptor key here. It is happening offline and we just want to get rid of unfinished transaction because it was abandoned anyways and we are also rewriting this value with generated one.

Reply:
Thanks @aliher1911! I'll remove the TODO.